docker/Makefile

```diff
@@ -242,6 +276,12 @@ package-fedora:
 	-n gerbil-$(BRANCH).fedora \
 	--description 'Gerbil Scheme Package' /opt/gerbil"
 
+package-opensuse:
+	docker run -v $(PWD):/src:z -t gerbil/opensuse:$(arch)-$(BRANCH) \
+	bash -c "zypper install -t package -y ruby ruby-devel rpm-build && \
+	gem install fpm && \
+	find /usr/lib64/ruby/gems -name fpm -type f -exec {} -s dir -p /src/ -t rpm -n gerbil-$(BRANCH).opensuse --description 'Gerbil Scheme Package' /opt/gerbil \;"
```

Comment:
I think BRANCH is wrong, and we should be using some version instead (e.g. from git describe --tags).

Reply:
BRANCH is an external variable passed as a parameter. e.g. `make opensuse BRANCH=v0.18.1`